app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt

```diff
@@ -80,6 +81,7 @@ import kotlin.math.min
 @Composable
 fun KeyboardKey(
     key: KeyItemC,
+    ghostKey: KeyItemC? = null,
```

Comment:
Make a comment on what this is, mainly that these are set in the keyboard generation code, and shouldn't be set in the definition files.